spec/faraday/response/logger_spec.rb

```diff
@@ -55,6 +55,26 @@
     end
   end
 
+  context 'when logger with program name' do
+    let(:logger) { Logger.new(string_io, progname: 'my_best_program') }
+
+    it 'logs with program name' do
+      conn.get '/hello'
+
+      expect(string_io.string).to match('-- my_best_program: request:')
+      expect(string_io.string).to match('-- my_best_program: response:')
+    end
+  end
+
+  context 'when logger without program name' do
+    it 'logs without program name' do
+      conn.get '/hello'
+
+      expect(string_io.string).to match('-- : request:')
+      expect(string_io.string).to match('-- : response:')
```

Comment:
Not a huge fan of the `:` at the beginning, but I guess it can't be helped. It doesn't seem like we're printing it so I assume this is default `Logger` behaviour